pkg/util/taints/taints.go

```diff
@@ -34,6 +34,10 @@ const (
 	UNTAINTED = "untainted"
 )
 
+const (
+	OutOfServiceTaint = "out-of-service"
```

Comment:
Add a comment for this const, including a reference to the KEP.

Reply:
Per https://github.com/kubernetes/enhancements/issues/2268#issuecomment-1059349170, I think we should use a registered taint and _not_ a private taint here.
(In other words, the KEP is proposing an invalid design - we should fix the KEP and then update the implementation to match).

Reply:
@sftim -- Do we need to do any specific code changes to make a taint registered or simply following the naming conventions and putting it in the doc should work?

Reply:
We probably should add the new taint to here: https://github.com/kubernetes/kubernetes/blob/v1.24.0-alpha.3/staging/src/k8s.io/api/core/v1/well_known_taints.go#L48

Reply:
Sure, we can submit a PR to add the new taint to the dev-1.24 branch of k/website.